src/social-card/templates/highlight-card.template.ts

```diff
@@ -15,7 +16,7 @@ const highlightCardTemplate = (
       <div tw="flex-row justify-between" style="gap: 16px;">
         <div tw="w-906px flex-col flex-nowrap" style="gap: -10px;">
           <h1 tw="text-72px leading-72px text-black tracking-tight" style="width: 926px; font-weight: 500;">
-               OpenSauced Highlight
+               OpenSauced Highlight 
```

Comment:
nit: Does this need a hanging space at the end of this line?

Reply:
No, I had an emoji here to test it (When I did not find an highlight with emojis :) ) 

Would adding a pizza emoji be a good option here?

Reply:
Sure! Good idea!! 🍕

Reply:
Thank you, I just added an emoji at the beginning of this line! 🍕